### PR TITLE
Fix wording

### PR DESCRIPTION
**Pull request by DennisBunea**

### CODE REVIEW

File: source/howto/code-repository/private-github-repository/index.rst [hunk 1]
```diff
@@ -44,9 +44,9 @@ This will generate two files:
 ..
 
 Add the public key to GitHub
-###################################
+###################################
```
Comment (noorula) [inline source/howto/code-repository/private-github-repository/index.rst:47]:
```suggestion
###################################
```

**Review by noorula — CHANGES_REQUESTED** [review]
Looking good, one small change.

**Comment by DrazenDodik** [comment]
- `github-key-2.png` shows adding a deploy key to GitHub. That looks like a private key to me, instead of a public key? The private key goes to Valohai (starts with ---BEGIN) and the public key goes to GitHub
- `github-key-3.png` is an old screenshot from a different repo. Here it shows ruksi/my-project. I'd update this screenshot.
- Looks like the screenshots on the Valohai UI haven't been updated. Can you check those? Example below: 
image
If you look at the code, it looks like these files are in a different location. I'd maybe move them to /howto/code-repository/, replace the images with new screenshots, and then update the code to point to new images.
```
.. image:: /tutorials/valohai-key-1.png
   :alt: Valohai - route to repository settings
``` 

I'd resize the images, because some of them are massive now and will cause slower loading times. For example `source/howto/code-repository/private-github-repository/github-key-2.png` used to be W: 999px | H: 656px and is now W: 3663px | H: 1541px

Do the same changes across GitHub/BitBucket/GitLab

**Review by noorula — APPROVED** [review]
Fix the email and the private key, then good to go.

**Review by noorula — APPROVED** [review]
Looks good!

**Pull request merged**